addons/payment/models/payment_acquirer.py

```diff
@@ -53,10 +54,12 @@ class PaymentAcquirer(models.Model):
     _order = 'website_published desc, sequence, name'
 
     name = fields.Char('Name', required=True, translate=True)
+    display = fields.Char('Display as', translate=True,
```

Comment:
Should be renamed into ``website_name``

Reply:
In fact the display_name should be the name. The technical name has already been moved into the field ``provider``. So you should simply change the name on the acquirers instead of adding a new field.

Reply:
the provider is not unique for every acquirer. There can be more acquirers with the same provider. By default there are already 2 with provider 'Wire Transfer'.